pkg/distro/fedora/images.go

```diff
@@ -481,6 +516,23 @@ func iotRawImage(workload workload.Workload,
 	}
 	img.OSName = "fedora-iot"
 
+	if !common.VersionLessThan(distro.Releasever(), "38") {
+		img.Ignition = true
+		switch img.Platform.GetImageFormat() {
+		case platform.FORMAT_RAW:
+			img.IgnitionPlatform = "metal"
+			if bpIgnition := customizations.GetIgnition(); bpIgnition != nil && bpIgnition.FirstBoot != nil && bpIgnition.FirstBoot.ProvisioningURL != "" {
+				img.KernelOptionsAppend = append(img.KernelOptionsAppend, "ignition.config.url="+bpIgnition.FirstBoot.ProvisioningURL)
+			}
+		case platform.FORMAT_QCOW2:
+			img.IgnitionPlatform = "qemu"
```

Comment:
Why doesn't this platform also get the `bpIgnition` that raw gets?

Reply:
Similar answer: the iot-qcow2 in the original PR didn't get the `ignition.config.url` option: https://github.com/paulwhalen/osbuild-composer/blob/e1d879691eb74bde29746e3dfa20447efad70b35/internal/distro/fedora/images.go#L501
(note that in that PR the image function was completely separate and here it's shared between the two image types).

Perhaps it was an oversight.  I'll defer to @runcom again.